packages/pluggableWidgets/datagrid-dropdown-filter-web/src/DatagridDropdownFilter.tsx

```diff
@@ -69,6 +70,16 @@ function getAttributeTypeErrorMessage(type?: string): string | null {
         : null;
 }
 
+function validateValues(listAttribute: ListAttributeValue, values: FilterOption[]): string | null {
+    if (values.length === 0) {
+        return null;
+    }
+
+    return !listAttribute.universe?.some(value => values.some(v => v.value === value))
```

Comment:
If we bail out early if the attribute has no universe, then you could write ` return values.some(v => !universe.includes(v.value)) ? ...`, which is easier to parse with the human eye :).

Reply:
Fixed